fvm/src/machine/engine.rs

```diff
@@ -121,7 +121,7 @@ pub fn default_wasmtime_config() -> wasmtime::Config {
     // wasmtime default: 512KiB
     // Set to something much higher than the instrumented limiter.
     // Note: This is in bytes, while the instrumented limit is in stack elements
-    c.max_wasm_stack(4 << 20).unwrap();
+    c.max_wasm_stack(4 << 20);
```

Comment:
Per [this](https://github.com/bytecodealliance/wasmtime/pull/4262/files#diff-908a422cef87524478865a23918af79040c970e02a3cc6862f6eb508710606f9R508), can only cause failures at `Engine` creation if set to 0, or async is enabled.